code/AuditHookManyManyList.php

```diff
@@ -11,6 +11,8 @@
 /**
  * AuditHookManyManyList is meant to override ManyManyList. When a Member is
  * removed from a Group, it logs the event.
+ *
+ * @extends ManyManyList<Member>
```

Comment:
This means it extends a ManyManyList that contains a list of Members correct?

However silverstripe/auditor will use injector swap out ALL ManyManyList with AuditHookManyManyList ... so `<Member>` could be anything

Is the assumption here that if you manually instantiate an AuditHookManyManyList it would only be for Members?

Reply:
Ahh. For some reason in my brain this was only replacing `ManyManyList` when it's explicitly a list of members, but as you pointed out that's not necessarily the case. I'll update to template T of dataobject.

Reply:
Done